package/network/ipv6/ds-lite/files/dslite.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
-# dslite.sh - IPv4-in-IPv6 tunnel backend
-# Copyright (c) 2013 OpenWrt.org
+# dslite.sh - IPv4-in-IPv6 tunnel backend for ipip6 and ds-lite
+# Copyright (c) 2024 OpenWrt.org
```

Comment:
I remember there were discussions about how this OpenWrt.org copyright isn't valid and we all stopped updating them at one point. You can leave this as-is.